test/unit/alignment/aligned_sequence_test_template.hpp

```diff
@@ -28,6 +28,7 @@ TYPED_TEST_P(aligned_sequence, fulfills_concept)
 {
     EXPECT_TRUE((AlignedSequence<TypeParam>));
     EXPECT_FALSE((AlignedSequence<std::vector<dna4>>));
+    EXPECT_TRUE((AlignedSequence<std::array<gapped<dna4>, 0>>));
```

Comment:
@smehringer As it turns out, it seems that this is not a aligned sequence. I'm not sure which condition of the concept fails.

Reply:
yes this is because I require to be able to use insert and erase on an aligned sequence.
But it does not matter because I do not require the AlignedSequence concept for writing an alignment (I need no insert functionality). I just require that the value_type  is comparable to gap. so it should be fine? What does the output file tests say?

Reply:
I added a strategy meeting card.